setup.js

```
@@ -0,0 +1,28 @@
+import 'dotenv/config';
+
+import { User } from './src/models/User.js';
+import { Token } from './src/models/Token.js';
+import { client } from './src/utils/db.js';
+
+async function initializeDatabase() {
+  // Guard sync with force: true to only run in test or development environments
+  if (
+    process.env.NODE_ENV !== 'test' &&
+    process.env.NODE_ENV !== 'development'
+  ) {
+    throw new Error(
+      'Database sync with force: true is not allowed in production environment',
+    );
+  }
+
+  try {
+    await client.sync({ force: true });
+  } catch (error) {
+    throw error;
+  }
```

The inner try/catch currently rethrows the same error (`catch (error) { throw error; }`) which is redundant. If you only want to bubble the error up, you can omit the catch. Otherwise, add logging or wrap the error with context before rethrowing. Also consider handling specific errors (e.g. connection/authentication failures) to give clearer messages.